app/modules/entities/components/layouts/entity_layout_actions.svelte

```diff
@@ -12,16 +12,15 @@
 
   let waitForEntityRefresh
 
-  const { uri, type, claims } = entity
+  const { uri, wdUri, type, claims } = entity
```

Comment:
After building the server types on `same-as` branch, there is still some TS errors:

```
/client/app/modules/entities/components/layouts/entity_layout_actions.svelte:15:16
Error: Property 'wdUri' does not exist on type 'SerializedEntity'. (ts)

  const { uri, wdUri, type, claims } = entity



/client/app/modules/entities/components/editor/entity_edit_menu.svelte:18:34
Error: Property 'invUri' does not exist on type 'SerializedEntity'. (ts)
  let flash
  const { uri, isWikidataEntity, invUri, wdUri, label, historyPathname } = entity
  const { hasDataadminAccess } = app.user


/client/app/modules/entities/components/editor/entity_edit_menu.svelte:18:42
Error: Property 'wdUri' does not exist on type 'SerializedEntity'. (ts)
  let flash
  const { uri, isWikidataEntity, invUri, wdUri, label, historyPathname } = entity
  const { hasDataadminAccess } = app.user


/client/app/modules/entities/components/wikidata_edit_intro.svelte:18:19
Error: Property 'id' does not exist on type 'SerializedWdEntity'. (ts)
    wikidataOauth = API.auth.oauth.wikidata + `&redirect=${editPathname}`
    wdId = entity.id
    wikiUrl = getWdWikiUrl(wdId)
```

Reply:
Fixed in 85b15c87d